### PR TITLE
DeprecationWarning on deprecated colors API

### PR DESCRIPTION
**Pull request by Carreau**
Ok, been deprecated since 2009.
We can probably remove it.

### CODE REVIEW

**Comment by Carreau** [comment]
WTF unrelated error reproductible ? Maybe becasue we test on Python 3.5.0.rc4, sent other PR to update that.

File: IPython/core/excolors.py [hunk 1]
```diff
@@ -128,8 +130,17 @@ def exception_colors():
     return ex_colors
 
 
+class Deprec(object):
+
+    def __init__(self, wrapped_obj):
+        self.wrapped=wrapped_obj
+
+    def __getattr__(self, name):
+        warnings.warn("Using ExceptionColors global is deprecated", UserWarning)
```
Comment (minrk) [inline IPython/core/excolors.py]: Should be a DeprecationWarning, presumably.

Comment (Carreau) [inline IPython/core/excolors.py]: I was unsure as DeprecationWarning are diable by default.

Comment (Carreau) [inline IPython/core/excolors.py]: (but Done)

**Pull request merged**